### DISCUSSION TITLE
建议增加模拟正版进入正版服务器功能

### DISCUSSION THREAD

**Discussion by ColorAWA** [Meloong-Git/PCL#1716] [讨论]
让离线玩家能玩hypixel

**Comment by iPodtouch6557** [comment]
首先，请将该建议交一个issue
而且，其实可行性不强……真的建议买一个正版，~虽然我也没钱~

**Comment by ColorAWA** [comment]
浮木不给买啊

**Comment by WForst-Breeze** [comment]
至少以我个人的认识上来看，离线模式无法做到模拟正版进入服务器这一功能 :(

**Reply by iPodtouch6557** [reply]
其实我就这个意思……你的个人信息、你的账号信息都哪里来，~难道借龙猫的用吗~

**Reply by WForst-Breeze** [reply]
退一万步讲，就算可以做，这也会严重侵犯正版玩家和相关服务器腐竹的权益（）因为这样的话作弊成本会很低，更何况这个功能做不出来 😶

**Reply by iPodtouch6557** [reply]
> 退一万步讲，就算可以做，这也会严重侵犯正版玩家和相关服务器腐竹的权益（）因为这样的话作弊成本会很低，更何况这个功能做不出来 😶

我就见过开不了正版验证的服务器声明正版问题，真的是权益问题……

**Comment by Chiloven945** [comment]
为什么这种评论会出现在github上，版权意识这么弱的吗？
